### PR TITLE
Define the ordering of click event and lostpointercapture

### PR DESCRIPTION
**Pull request by NavidZ**
@RByers @teddink @patrickkettner

Add a note related to #75.
Also the associated test is w3c/pointerevents#4763

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -853,6 +853,7 @@
         <section>
             <h3>Implicit Release of Pointer Capture</h3>
             <p>Immediately after firing the <code>pointerup</code> or <code>pointercancel</code> events, a user agent MUST clear the <a>pending pointer capture target override</a> for the <code>pointerId</code> of the <code>pointerup</code> or <code>pointercancel</code> event that was just dispatched, and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code> if necessary.</p>
+            <div class="note">If user agent supports firing <code>click</code> event (see <a title="compatibility mouse events" href="#dfn-compatibility-mouse-events">compatibility mouse events</a>) and if in an implicit release scenario both <code>click</code> and <code>lostpointercapture</code> events are fired, <code>click</code> should precede <code>lostpointercapture</code>.</div>
```
Comment (RByers) [inline index.html]: Wording seems fine to me.  However, since we're testing this, it should probably technically be normative - i.e. not inside a 'note'.

**Comment by NavidZ** [comment]
@RByers does this look good?

File: index.html [hunk 2]
```diff
@@ -853,6 +853,7 @@
         <section>
             <h3>Implicit Release of Pointer Capture</h3>
             <p>Immediately after firing the <code>pointerup</code> or <code>pointercancel</code> events, a user agent MUST clear the <a>pending pointer capture target override</a> for the <code>pointerId</code> of the <code>pointerup</code> or <code>pointercancel</code> event that was just dispatched, and then run <a href="#process-pending-pointer-capture">Process Pending Pointer Capture</a> steps to fire <code>lostpointercapture</code> if necessary.</p>
+            <p>If user agent supports firing <code>click</code> event (see <a title="compatibility mouse events" href="#dfn-compatibility-mouse-events">compatibility mouse events</a>) and if in an implicit release scenario both <code>click</code> and <code>lostpointercapture</code> events are fired, <code>click</code> should precede <code>lostpointercapture</code>.</p>
```
Comment (patrickhlauke) [inline index.html]: "If *the* user agent supports ..."

also, while "precede" is correct, maybe using simpler language like "be fired before" may make it more understandable?

File: index.html [hunk 3]
```diff
@@ -853,7 +853,7 @@
...
-            <p>If user agent supports firing <code>click</code> event (see <a title="compatibility mouse events" href="#dfn-compatibility-mouse-events">compatibility mouse events</a>) and if in an implicit release scenario both <code>click</code> and <code>lostpointercapture</code> events are fired, <code>click</code> should precede <code>lostpointercapture</code>.</p>
+            <p>If the user agent supports firing <code>click</code> event (see <a title="compatibility mouse events" href="#dfn-compatibility-mouse-events">compatibility mouse events</a>) and if in an implicit release scenario both <code>click</code> and <code>lostpointercapture</code> events are fired, <code>click</code> should be fired before <code>lostpointercapture</code>.</p>
```
Comment (patrickhlauke) [inline index.html]: sorry to be a pain, missed it on first reading: also needs a "the" in "supports firing *the* click event"

Comment (NavidZ) [reply index.html]: No Worries. Thanks for catching it.

**Comment by patrickhlauke** [comment]
LGTM thanks for indulging me :)

**Pull request merged**